generator/config/query/bitsAllSet.yaml

```diff
@@ -12,4 +12,29 @@ arguments:
         type:
             - int
             - binData
-            - array # of int
+        variadic: array
+tests:
+    -
+        name: 'Bit Position Array'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/query/bitsAllSet/#bit-position-array'
+        pipeline:
+            -
+                $match:
+                    a:
+                        $bitsAllSet: [1, 5]
+    -
+        name: 'Integer Bitmask'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/query/bitsAllSet/#integer-bitmask'
+        pipeline:
+            - $match:
+                  a:
+                      # $bitsAllSet: 50
+                      $bitsAllSet: [50]
+    -
+        name: 'BinData Bitmask'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/query/bitsAllSet/#bindata-bitmask'
+        pipeline:
+            - $match:
+                  a:
+                      $bitsAllSet:
+                          - { $binary: { base64: 'TUM9PQ==', subType: '00' } }
```

Comment:
https://www.mongodb.com/docs/manual/reference/operator/query/bitsAllSet/#bindata-bitmask uses `MC==`, which also isn't canonical:

```
> BinData(0, "MC==")
Binary.createFromBase64("MA==", 0)
```

I'd suggest including this in your DOCSP ticket as well. I've no idea where the author(s) got these values from.